index.html

```diff
@@ -35,5 +35,149 @@ <h1 class="headline">I'm Didier<br/>Glad you're here!</h1>
+                    <h2 class="work-title">Tonic</h2>
+                    <div class="work-subtitle">
+                        <p class="company">Canopy</p>
+                        <img src="images/counter.png" width="10" height="10" alt="counter">
+                        <p>Back End Dev</p>
+                        <img src="images/counter.png" alt="counter">
+                        <p>2015</p>
+                    </div>
+                    <p class="work-description">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    <ul class="tags">
+                        <li class="tag">html</li>
+                        <li class="tag">css</li>
+                        <li class="tag">javascript</li>
+                    </ul>
+                    <a href="#" class="work-link">See Project</a>
+                </div> 
+            </div>
+        </div>       
+    </section>
+    <section class="about-section">
+        <div class="about">
+            <h2 class="about-title">About<br/> Myself</h2>
+            <p class="about-description">Hello I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+            <div>
+                <p class="connect">Let's connect</p>
+                <ul class="social-links">
+                    <li><a href="https://www.twitter.com/didierganthier_"><img src="images/twitter.svg" alt="twitter"></a></li>
+                    <li><a href="https://www.linkedin.com/in/didierganthier"><img src="images/linkedin.svg" alt="linkedin"></a></li> 
+                    <li><a href="https://www.medium.com/@didierganthier"><img src="images/medium.svg" alt="medium"></a></li>
+                    <li><a href="https://wwww.github.com/didierganthier"><img src="images/github.svg" alt="github"></a></li>
+                </ul>
+            </div>
+            <a href="#" class="about-work-link">Get My Resume</a>
+        </div>
+        <div class="languages">
+            <div class="languages-head">
+                <h3 class="languages-title">Languages</h3>
+                <img src="images/arrow-down.png" width="25" height="25" alt="arrow-down">
+            </div>
+            <div class="language-container">
+                <div class="language-card">
+                    <img src="images/javascript.png" alt="javascript">
+                    <p class="language-name">Javascript</p>
+                </div>
+                <div class="language-card">
+                    <img src="images/html.png" alt="html">
+                    <p class="language-name">HTML</p>
+                </div>
+                <div class="language-card">
+                    <img src="images/css.png" alt="css">
+                    <p class="language-name">CSS</p>
+                </div>
+            </div>
```

Comment:
- [ ] Good job so far 👏 🔥 You have done a great job so far. However, from the markup guidelines in this section, the Technology categories (i.e: "Languages", "Frameworks" and "Skills") are supposed to be in `<ul>` and each item in `<li>` and also the technology in each category is supposed to be a list too `<ul>` and each of them being in `<li>` tag. In other words, **you are supposed to implement the nested lists** Please refer to this [aticle](https://www.webucator.com/article/how-to-properly-nest-lists-in-html/) to learn how to implement them

**HINT**
```
<ul>
  <li>Fruit
    <ul>
      <li>Bananas</li>
      <li>Apples</li>
      <li>Pears</li>
    </ul>
  </li>
  <li>Vegetables</li>
  <li>Meat</li>
</ul>
```